templates/azuredeployUI.json

```diff
@@ -0,0 +1,246 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2021-09-09/uiFormDefinition.schema.json",
+    "view": {
+        "kind": "Form",
+        "properties": {
+            "title": "IoT Hub Portal Deployment",
+            "steps": [
+                {
+                    "name": "basics",
+                    "label": "Basics",
+                    "elements": [
+                        {
+                            "name": "resourceScope",
+                            "type": "Microsoft.Common.ResourceScope",
+                            "location": {
+                                "resourceTypes": [
+                                    "microsoft.resources/resourcegroups"
+                                ]
+                            }
+                        }
+                    ]
+                },
+				{
+					"name": "commonStep",
+					"label": "Common Settings",
+					"elements": [
+						{
+							"name": "uniqueSolutionPrefix",
+							"type": "Microsoft.Common.TextBox",
```

Comment:
Should validate the input : 
- only alpha numeric characters 
- max 16 characters